testing/sdk_utils.py

```diff
@@ -120,6 +121,18 @@ def is_strict_mode():
     return os.environ.get('SECURITY', '') == 'strict'
 
 
+def is_channel(channel):
```

Comment:
This is good, but changes here would cause all frameworks to be rebuilt. I think we could use `dcos_version > 1.11` as a substitute.

Reply:
I agree, that's better.